index.d.ts

```diff
@@ -0,0 +1,9 @@
+type ClassValue = string | number | ClassDictionary | ClassArray;
+
+interface ClassDictionary {
+	[id: string]: boolean;
```

Comment:
same here, this needs to allow `undefined` and `null`. also, personal preference (generally in the TS community), can you use 4-space indentation instead of tabs?